main.go

```diff
@@ -537,19 +537,39 @@ func main() {
 	}
 
 	// Simulator infos
-	simulator, err := xcodeutil.GetSimulator(configs.SimulatorPlatform, configs.SimulatorDevice, configs.SimulatorOsVersion)
-	if err != nil {
+	var sim simulator.InfoModel
+	var osVersion string
+	var errGetSimulator error
+	if configs.SimulatorOsVersion == "latest" {
+		simulatorPlatformSplit := strings.Split(configs.SimulatorPlatform, " Simulator")
+		if len(simulatorPlatformSplit) == 0 {
+			errGetSimulator = fmt.Errorf("failed to parse simulator platform (%s)", configs.SimulatorPlatform)
+		}
+
+		var simulatorDevice = configs.SimulatorDevice
```

Comment:
this could go up to the batch var decls block with the default value configs.SimulatorDevice so this if true block will only override but everything uses a common var